tests/ttnn/unit_tests/operations/test_creation.py

```diff
@@ -115,6 +158,32 @@ def test_full_like(device, input_shape, fill_value):
     assert torch.allclose(torch_output_tensor, output_tensor)
 
 
+@pytest.mark.parametrize(
+    "input_shape",
+    [
+        [32, 32],
+        [5, 96, 64],
```

Comment:
Lets test shapes in rank 2-5 today.

Later, once tensor layout work is complete - we will have to start testing with shapes of rank 0 to 5 and additionally for 0 volume.

Reply:
@ayerofieiev-tt yeah sure. I've added shapes with rank 2-5